lib/features/identity_creator/domain/model/identity_cache.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:tmail_ui_user/features/manage_account/presentation/model/identity_action_type.dart';
+import 'package:tmail_ui_user/features/public_asset/domain/model/public_assets_in_identity_arguments.dart';
+
+class IdentityCache with EquatableMixin {
```

Comment:
more clearly also this model too

Reply:
It is also for caching identity content.